jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryResultsTable.kt

```diff
@@ -58,5 +61,19 @@ class QueryResultsTable(
             }
         }
     }
+
+    private fun installDetailedLogRecordOpenListener() {
+        object : DoubleClickListener() {
+            override fun onDoubleClick(e: MouseEvent): Boolean {
+                // assume you can't double click multiple selection
+                val identifier = resultsTable.selectedObject?.ptr() ?: return false
+                val window = QueryResultsWindow.getInstance(project)
```

Comment:
any reason not to do this in the `launch`?

Reply:
feels the same to me, but probably better in the `launch`